test/libsolidity/syntaxTests/types/strict_explicit.sol

```diff
@@ -0,0 +1,42 @@
+contract B{}
+contract C
+{
+    function f() public pure {
+
+        uint16 a = uint16(uint8(int8(-1)));
+        a;
```

Comment:
Why these?

Reply:
I wanted to avoid the unused local variable warning.

Reply:
ah ok